deployer/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM openshift/origin:v1.2.0
+FROM openshift/origin:v1.5.0-alpha.2
```

Comment:
Do we really want to bump this far?  Might it be more prudent to use v1.4.0 or v1.4.1?  Maybe this is less important at the moment now that we are moving away from the deployer